src/containers/singleEvent/index.tsx

```diff
@@ -125,13 +142,16 @@ const SingleEvent: React.FC<SingleEventProps> = ({ events }) => {
                 <GrayButton to={BASE_EVENTS_ROUTE + id}>View RSVP</GrayButton>
               </AdminActionButtonList>
             )}
-            <EventDetails {...event[0]} />
+            <EventDetails
+              {...event[0]}
+              announcements={eventAnnouncements.result}
+            />
           </>
         );
       } else {
         return <p>That event does not exist!</p>;
       }
-    } else if (asyncRequestIsFailed(eventsList)) {
+    } else if (asyncRequestIsFailed(events)) {
       return <p>The request failed.</p>;
```

Comment:
🌱 We should ultimately display the error message returned by the API in most cases here, but we can wait on that until I finish the ApiError interface and utils.